dotnetv3/Dockerfile

```diff
@@ -1,18 +1,12 @@
 # syntax=docker/dockerfile:1
-# Status: Beta
-# GA updates: https://github.com/awsdocs/aws-doc-sdk-examples/issues/4126 
 FROM mcr.microsoft.com/dotnet/sdk:6.0
 
-# Enable the .NET 5.0 SDK if running 5.0 examples:
+# Copy source code
+COPY . /dotnetv3/
+
+# Perform build steps
 COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/sdk /usr/share/dotnet/sdk
 COPY --from=mcr.microsoft.com/dotnet/sdk:5.0 /usr/share/dotnet/shared /usr/share/dotnet/shared
+RUN dotnet build
```

Comment:
I tested this docker file and it no longer is able to build without the cd /dotnetv3 in the command. The changes also removed the folder parameter from the test command, which will prevent the tests from running. I think we can remove the comments if you want, but the commands we had need to stay the same for it to run.

Reply:
OK I will add this line back as `RUN cd /dotnetv3 && dotnet build`! Thanks Rachel.

Reply:
This specific line is actually causing an error no matter how you run it: 

```
109.4 /dotnetv3/Cognito/Tests/CognitoWrapperTests.cs(131,45): error CS1061: 'CognitoWrapper' does not contain a definition for 'RespondToAuthChallengeAsync' and no accessible extension method 'RespondToAuthChallengeAsync' accepting a first argument of type 'CognitoWrapper' could be found (are you missing a using directive or an assembly reference?) [/dotnetv3/Cognito/Tests/CognitoTests.csproj]
109.4 /usr/share/dotnet/sdk/6.0.414/Microsoft.Common.CurrentVersion.targets(5097,5): error MSB3030: Could not copy the file "/dotnetv3/EC2/VirtualPrivateCloudExamples/CreateVPCforS3Example/appsettings.local.json" because it was not found. [/dotnetv3/EC2/VirtualPrivateCloudExamples/CreateVPCforS3Example/CreateVPCforS3Example.csproj]
```

Until this is fixed, I'm going to comment out the build step.

Reply:
Have you tried it since PR 5447 was merged? That should resolve the issue above.

Reply:
Rebasing w/ https://github.com/awsdocs/aws-doc-sdk-examples/pull/5447

Reply:
The test command is still missing the folder parameter. I don't think anything needed to change except the comments saying it was Beta, is the correct?

Reply:
Sorry Rachel, yes, you're right. Reverted as requested.